src/Scene/Description/Style.js

```diff
@@ -1,154 +0,0 @@
-/**
```

Comment:
I think this class was recently introduczed, and it's going to be used soon.
The same goes for `Palette.js`, `StyleManager.js` and maybe `Description.js`.

@gchoqueux can you confirm?

Note that this is the 2nd time someone tries to remove it so it confirms that introducing *soon-to-be-used* code is a baaaad idea.